CONTRIBUTORS.md

```diff
@@ -0,0 +1,68 @@
+| Jeremy Soller          | jeremy@system76.com              | [@jackpot51](https://github.com/jackpot51)           |
+| Kathappan Esakkithevar | kathappan.esakkithevar@intel.com | [@kesakkit](https://github.com/kesakkit)             |
+| Ken Yao                | ken.yao@amd.com                  | [@YSHRong](https://github.com/YSHRong)               |
+| Leif Lindholm          | quic_llindhol@quicinc.com        | [@leiflindholm](https://github.com/leiflindholm)     |
+| Liming Gao             | gaoliming@byosoft.com.cn         | [@lgao4](https://github.com/lgao4)                   |
+| Ling Jia               | jialing@phytium.com.cn           |                                                      |
+| Marcin Juszkiewicz     | marcin.juszkiewicz@linaro.org    | [@hrw](https://github.com/hrw)                       |
+| Marcin Wojtas          | marcin.s.wojtas@gmail.com        | [@wojtas-marcin](https://github.com/wojtas-marcin)   |
+| Marvin Häuser          | mhaeuser@posteo.de               | [@mhaeuser](https://github.com/mhaeuser)             |
+| Meenakshi Aggarwal     | meenakshi.aggarwal@nxp.com       | [@Meenakshi88](https://github.com/Meenakshi88)       |
+| meng-cz                | mengcz1126@gmail.com             | [@meng-cz](https://github.com/meng-cz)               |
+| Michael D Kinney       | michael.d.kinney@intel.com       | [@mdkinney](https://github.com/mdkinney)             |
+| Narinder Dhillon       | ndhillon@marvell.com             | [@ndhillonm](https://github.com/ndhillonm)           |
+| Nate DeSimone          | nathaniel.l.desimone@intel.com   | [@nate-desimone](https://github.com/nate-desimone)   |
+| Nhi Pham               | nhi@os.amperecomputing.com       | [@nhivp](https://github.com/nhivp)                   |
+| Nickle Wang            | nicklew@nvidia.com               | [@nicklela](https://github.com/nicklela)             |
+| Paul Grimes            | paul.grimes@amd.com              | [@pbgrimes](https://github.com/pbgrimes)             |
+| Pedro Falcato          | pedro.falcato@gmail.com          | [@heatd](https://github.com/heatd)                   |
+| Peng Xie               | xiepeng@phytium.com.cn           |                                                      |
+| Ray Ni                 | ray.ni@intel.com                 | [@niruiyu](https://github.com/niruiyu)               |
+| Rebecca Cran           | rebecca@os.amperecomputing.com   | [@bcran](https://github.com/bcran)                   |
+| Rosen Chuang           | rosen.chuang@intel.com           | [@rbitw](https://github.com/rbitw)                   |
+| Sai Chaganty           | rangasai.v.chaganty@intel.com    | [@SaiChaganty](https://github.com/SaiChaganty)       |
+| Saloni Kasbekar        | saloni.kasbekar@intel.com        | [@SaloniKasbekar](https://github.com/SaloniKasbekar) |
+| Sami Mujawar           | sami.mujawar@arm.com             | [@samimujawar](https://github.com/)                  |

```

Comment:
I am not sure what the last column means, but looking at other entries should this be as below
- | Sami Mujawar | sami.mujawar@arm.com | [@samimujawar](https://github.com/) |
+ | Sami Mujawar | sami.mujawar@arm.com | [@samimujawar](https://github.com/samimujawar) |